addons/kommander/1.1.0/kommander-48.yaml

```diff
@@ -0,0 +1,102 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-47"
```

Comment:
this need to be bumped to `-48`, otherwise lgtm!

Reply:
Added a commit to do this- thanks